src/models/github.js

```diff
@@ -11,6 +11,22 @@ class GitHub extends Fetcher {
     return this.get('notifications')
   }
 
+  getPandaRocketTasks() {
+    const urlPath = 'search/issues?q=team=identity-team-panda-rocket'
```

Comment:
What about storing `identity-team-panda-rocket` in the JSON config file for now, as a more temporary way of specifying the team? This method could be called `getTeamTasks` and take a `teamName` that it passes in the query.